lib/Request.js

```diff
@@ -179,22 +179,31 @@ class Request extends CallableInstance {
 
   async _setup (opts = {}) {
     try {
-      const connection = await this._remit._connection
-      this._channel = await connection.createChannel()
+      if (!this._remit._publishChannels[opts.event]) {
+        const publishChannelP = this._remit._publishChannels[opts.event] = (async () => {
+          const connection = await this._remit._connection
+          return connection.createChannel()
+        })()
+
+        const publishChannel = await publishChannelP
+        
+        await publishChannel.consume(
+          'amq.rabbitmq.reply-to',
+          bubble.bind(this._incoming.bind(this)),
```

Comment:
So using `this._incoming` here is interesting.

Seeing as this is going to be used by multiple requests, let's move `_incoming` to the `Remit` class and instead use `bubble.bind(this._remit._incoming.bind(this._remit))`.